src/melee.cpp

```diff
@@ -2273,6 +2251,39 @@ std::vector<special_attack> player::mutation_attacks(Creature &t) const
 
 std::string melee_message( const ma_technique &tec, player &p, const dealt_damage_instance &ddi )
 {
+    // Those could be extracted to a json
+
+    // Three last values are for low damage
+    static const std::array<std::string, 6> player_stab = {{
+        "You impale %s", "You gouge %s", "You run %s through",
```

Comment:
What is the proper way of doing that? Would translating them inside the arrays cause any unspecified behavior or be executed before translating system is set up or anything like that?
I recall `_( str.c_str() )` being used in iuse_actor.cpp - what's the difference between that and those lines?